[7.0] Group by HIP Stream in Perfetto output (#306)
ROCm/rocprofiler-systems: docs/how-to/configuring-runtime-options.rst

   ROCPROFSYS_ROCM_EVENTS = GPUBusy     SQ_WAVES:device=0    SQ_INSTS_VALU:device=1

ROCPROFSYS_ROCM_GROUP_BY_QUEUE
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> By default, Perfetto trace will show the HIP streams to which kernel

Reviewer: ```suggestion
By default, Perfetto trace shows the HIP streams to which the kernel
```

> and memory copy operations submitted. With the

Reviewer: ```suggestion
and memory copy operations are submitted. However, with the
```

> ``ROCPROFSYS_ROCM_GROUP_BY_QUEUE=ON`` setting, the trace will display HSA queues

Reviewer: ```suggestion
``ROCPROFSYS_ROCM_GROUP_BY_QUEUE=ON`` setting, the trace will rather display HSA queues
```

> to which these kernel and memory operations were submitted.

Reviewer: ```suggestion
to which these kernel and memory operations are submitted.
```

> to which these kernel and memory operations are submitted.

Author: ```suggestion
By default, the Perfetto trace groups the kernel dispatch and memory copy operations 
to tracks corresponding to their HIP Stream ID. However, with the
``ROCPROFSYS_ROCM_GROUP_BY_QUEUE=ON`` setting, the events are on separate tracks and grouped by hardware queue.
```
